### DISCUSSION TITLE
Adding Linux capabilities to a BuildConfig container

### DISCUSSION THREAD

**Discussion by bpaskin** [okd-project/okd#1747] [Q&A] [answered]
Hi,

I have a `BuildConifg` that requires several Linux capabilities.  I have the `builder` service account tied to an SCC that has the following capabilities added:
```
defaultAddCapabilities:
- CHECKPOINT_RESTORE
- SETPCAP
- SYS_PTRACE
```

However, when the container is running and trying to build the image it is missing the capabilities:

```
Bounding set =cap_chown,cap_dac_override,cap_fowner,cap_fsetid,cap_kill,cap_setgid,cap_setuid,cap_setpcap,cap_net_bind_service,cap_sys_chroot,cap_setfcap
```

Is it possible to enable these capabilities in a `Build`?

Thanks, Brian

**Comment by vrutkovs** [comment, marked as answer]
I don't think we have a configuration for that (in fact, Builds are usually moving into reducing the used capabilities). You should file a [new RFE via Customer Portal](https://access.redhat.com/support/cases/new)

**Comment by bpaskin** [comment]
Thanks, I opened an RFE.
